examples/filters/pycolorfilter/color_filter.py

```diff
@@ -0,0 +1,77 @@
+from typing import ClassVar, Mapping, Sequence, Optional, cast, Tuple, List, Any, Dict
+
+from typing_extensions import Self
+
+from viam.module.types import Reconfigurable
+from viam.proto.app.robot import ComponentConfig
+from viam.proto.common import ResourceName, ResponseMetadata, Geometry
+from viam.components.camera import Camera
+from viam.resource.types import Model, ModelFamily
+from viam.resource.base import ResourceBase
+from viam.media.video import NamedImage
+from PIL import Image
+from viam.errors import NoCaptureToStoreError
+from viam.services.vision import Vision
+
+class ColorFilterCam(Camera, Reconfigurable):
+    # A ColorFilterCam wraps the underlying camera `actual_cam` and only keeps the data captured on the actual camera if `vision_service`
```

Comment:
If you use triple quotes `"""` for the comment, it will show up in the doc string, which is probably what you want for this comment that specifies what's happening within this class